additional_cmd.txt

```diff
@@ -0,0 +1,28 @@
+
+# Make sure redhat user directory contents ownership is correct
+chown -R redhat:redhat /home/redhat/
+
+# Add the pull secret to CRI-O and set root user-only read/write permissions
+cat > /etc/crio/openshift-pull-secret << EOF
+{"auths":{"cloud.openshift.com":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K29jbV9hY2Nlc3NfZjExNThkNGM2NzgxNDczZDk3ZDQ3NTE2NTA4ZjFkYWE6M1dYRlM4M08wQjBBSjZCWUpBWUFZUzFZWDM3WEYwQ1QxQTEwREFWOVpPRzY0WU5GRDI4VFNZTEYzVUVCWEdRSA==","email":"vpunj@redhat.com"},"quay.io":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K29jbV9hY2Nlc3NfZjExNThkNGM2NzgxNDczZDk3ZDQ3NTE2NTA4ZjFkYWE6M1dYRlM4M08wQjBBSjZCWUpBWUFZUzFZWDM3WEYwQ1QxQTEwREFWOVpPRzY0WU5GRDI4VFNZTEYzVUVCWEdRSA==","email":"vpunj@redhat.com"},"registry.connect.redhat.com":{"auth":"fHVoYy1wb29sLWU3OWFkNDVjLTc1N2YtNGMzZi04YWIwLTUwYWM0OTE5OGM0MTpleUpoYkdjaU9pSlNVelV4TWlKOS5leUp6ZFdJaU9pSTVZVEUyTUdJME5UaG1PRFkwT1dVME9ERXpNalUzWldZME9UWXpZbUpqTWlKOS50T2EwWUtIVUVCX1ExcldXaFZuSTNJUU5BRlA2XzRXYnlYU2tqamQ1cEttM0lxd0JVdDJ6RDQzUk80VC01OUlQV2lmSjdVa2ZhcTRELUJ4bnRPMWFDRHo4bmU4c0ZvbEF2VWRtUE1RTkU5WVRZRVJCcXRXdEZkUmk4eF9vaFduSDdNcHVjM0hxam5TR3pCakt5ZEV0MHdacDN5UElpbk5RYVBGbXgwVzZDcFlZYVRWQjd5cVlDNjREVUtMSGNEZUxQTV9VMV9HTDdPb1A1Q3Q0V3J1Wmx3MHluVm1rZjhINWExM0g5bnRoaTdrQ1dLa0ROWXJYbkdta3JQb3VVSG5Razh1QUNCNWJ4Q0JtTWN5ejF1cFk3eWR1UGI2cXhQaWFjb0VUU2JaaEMwRnFMbnJjMmhsaHgtV2k3XzEtSTd1OTJIWkY0emdtNTEya0Q3QXIwMkgzMXFyRG5wdy15UmFpTERONDFXUDBaaDFJOVMwLThSZFdpQWcyWHdhYWZoRXhKTGRWMlpQZWI1LThyU0xfQnRnNXJ0ZWZTdFVzSHZHdTduZ3NvczdJOVNUWUVrUFFOVTE4d2JYSVVDMm4wdnJpZnBTaFdwOWhyblYza2RfWXh2a3dDUVhvekNrVWh3X0ExYUhkdnFvXzdRdHVGUEhETFR3bV9CdE5vWTRpUmpBb2h6SWpDY2ZmTl9SanJsTWM0Zjh5elR4M0VyelB4RExneUZBaTlLRWxHRThJVFFablBTdU1POFJkVjZnYlRBcU5OTktZVnlVQnEyQnlELTZJazRLakpoQV9vZEIzdFNDcmNLaWZZR3RLMEh4c1pJeDJ6VEtib1V1YmlZd2FheHZqSWtqOFZCOXg3c2x0Y2Zrdzk0UU9Xelg3RXB5MURBOE1ZVHY5eW0xOGd0cw==","email":"vpunj@redhat.com"},"registry.redhat.io":{"auth":"fHVoYy1wb29sLWU3OWFkNDVjLTc1N2YtNGMzZi04YWIwLTUwYWM0OTE5OGM0MTpleUpoYkdjaU9pSlNVelV4TWlKOS5leUp6ZFdJaU9pSTVZVEUyTUdJME5UaG1PRFkwT1dVME9ERXpNalUzWldZME9UWXpZbUpqTWlKOS50T2EwWUtIVUVCX1ExcldXaFZuSTNJUU5BRlA2XzRXYnlYU2tqamQ1cEttM0lxd0JVdDJ6RDQzUk80VC01OUlQV2lmSjdVa2ZhcTRELUJ4bnRPMWFDRHo4bmU4c0ZvbEF2VWRtUE1RTkU5WVRZRVJCcXRXdEZkUmk4eF9vaFduSDdNcHVjM0hxam5TR3pCakt5ZEV0MHdacDN5UElpbk5RYVBGbXgwVzZDcFlZYVRWQjd5cVlDNjREVUtMSGNEZUxQTV9VMV9HTDdPb1A1Q3Q0V3J1Wmx3MHluVm1rZjhINWExM0g5bnRoaTdrQ1dLa0ROWXJYbkdta3JQb3VVSG5Razh1QUNCNWJ4Q0JtTWN5ejF1cFk3eWR1UGI2cXhQaWFjb0VUU2JaaEMwRnFMbnJjMmhsaHgtV2k3XzEtSTd1OTJIWkY0emdtNTEya0Q3QXIwMkgzMXFyRG5wdy15UmFpTERONDFXUDBaaDFJOVMwLThSZFdpQWcyWHdhYWZoRXhKTGRWMlpQZWI1LThyU0xfQnRnNXJ0ZWZTdFVzSHZHdTduZ3NvczdJOVNUWUVrUFFOVTE4d2JYSVVDMm4wdnJpZnBTaFdwOWhyblYza2RfWXh2a3dDUVhvekNrVWh3X0ExYUhkdnFvXzdRdHVGUEhETFR3bV9CdE5vWTRpUmpBb2h6SWpDY2ZmTl9SanJsTWM0Zjh5elR4M0VyelB4RExneUZBaTlLRWxHRThJVFFablBTdU1POFJkVjZnYlRBcU5OTktZVnlVQnEyQnlELTZJazRLakpoQV9vZEIzdFNDcmNLaWZZR3RLMEh4c1pJeDJ6VEtib1V1YmlZd2FheHZqSWtqOFZCOXg3c2x0Y2Zrdzk0UU9Xelg3RXB5MURBOE1ZVHY5eW0xOGd0cw==","email":"vpunj@redhat.com"}}}
```

Comment:
@vrindle Please remove this immediately!!
cc @bn222  if you can override this PR.

Reply:
@wizhaoredhat sorry about that. 
I forgot! I removed it